### PR TITLE
Upgrade webpack & babel & eslint packages

### PR DESCRIPTION
**Pull request by donbot-inc**

### CODE REVIEW

**Comment by artem-malko** [comment]
By the way, something strange happen with NodeJS 9 in Travis(

**Comment by artem-malko** [comment]
And again, you are awesome!)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tars-cli",
-  "version": "1.9.8",
+  "version": "1.9.9",
```
Comment (artem-malko) [inline package.json]: I think, it has to be 1.10.0

**Comment by artem-malko** [comment]
So, fix version, wait for all tests are passed and I will merge it and publish.
May be it might be good idea to switch NodeJS 9 from CI?

File: package.json (hunk 1 above)
Comment (donbot-inc) [reply package.json]: Fixed in eb5a3ff

**Comment by donbot-inc** [comment]
> Something strange happen with NodeJS 9 in Travis(

Yes, and I have no idea how to fix it at the moment.

> May be it might be good idea to switch NodeJS 9 from CI?

You suggest to temporarily remove NodeJS v9 from the travis-ci config?

**Comment by artem-malko** [comment]
Yes, if there is come problems in travis with NodeJS 9 — it is not a reason to freeze release) It is working locally)

**Pull request merged**